linera-client/src/client_options.rs

```diff
@@ -455,6 +455,16 @@ pub enum ClientCommand {
         chain_id: Option<ChainId>,
     },
 
+    /// Synchronizes a validator with the local state of chains.
+    SyncValidator {
+        /// The public key of the validator to synchronize.
+        name: ValidatorName,
```

Comment:
Would it make sense to make this optional? If not specified, just try to update all validators?

Reply:
I think it would be useful, but I left it for later. I opened #3259 to discuss and track it.